_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,68 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector as an Integrator. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
+
+## Input for creating an IdentityAttribute
+
+To create an IdentityAttribute as an Integrator for your own Connector, you need to proceed as described in the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case documentation using the following content:
+
+| Property    | Value                                                                                                    |
+| ----------- | -------------------------------------------------------------------------------------------------------- |
+| `@type`     | `"IdentityAttribute"`                                                                                    |
+| `owner`     | `"<Address of your Connector>"`                                                                          |
+| `validFrom` | `"<start of Attribute validity>"`                                                                        |
+| `validTo`   | `"<end of Attribute validity>"`                                                                          |
+| `value`     | Specify an [IdentityAttribute Value]({% link _docs_integrate/attribute-values.md %}#identity-attributes) |
+| `tags`      | `["<additional information 1>", ..., "<additional information m>"]`                                      |
+
+You need to replace the placeholders marked with `<...>` appropriately. In particular, it is necessary that you insert one of the available [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the `value` property. You are not allowed to specify the Address of a Connector other than your own as the value for the `owner` property, as the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case relates to the creation of Attributes for yourself and not for others. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.
```

Comment:
> as the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case relates to the creation of Attributes for yourself and not for others
then I have a problem. Why do we need this page and also the other page if they actually do the same?

Reply:
I'm not sure if I'm the right person to answer this question in principle, as I didn't originally set up the various scenarios and use cases. But as I understand it, the purpose of the scenarios is to explain how to apply certain use cases. In this scenario, the "Create an Attribute" use case plays a central role. However, you also have the opportunity on this scenario page to mention how you can obtain the required Address of the Connector (use case "Get currently used Identity"), which Attribute Values you can use and where you can find them, or the differences in Attribute creation between simple and complex Attributes.

So I understand this scenario page as a way to connect information that supports someone in applying the use case and that goes beyond the technical documentation of the use case.

Reply:
That's good! But I think we have a naming clash if we have to explain this here.

Reply:
I have now seen in the runtime code that the use cases have been restructured/renamed and the "CreateAttribute" use case here should be replaced by the "CreateIdentityAttribute" use case. From this perspective, do you think it is a naming clash that the use case and the scenario have the same name, and if so, how would you recommend changing this issue?

Reply:
Yep, it was changed. But it still only allows to create own IdentityAttributes.

Reply:
I've changed the sentence now so that it's clear that only own IdentityAttributes can be created.

As the documentation for the "Create an attribute" use case will change with #179 and this scenario page now already refers to the new documentation/functionality, it may be necessary to wait until #179 is published before merging this scenario